modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogObjectDescriptor.java

```diff
@@ -59,10 +59,11 @@ public String toString() {
     }
 
     /** Catalog object type. */
-    enum Type {
+    public enum Type {
```

Comment:
why do you change this?

Reply:
Because this package private type is exposed via public accessor method `type()`.